src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,36 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#divide-program-logic): Divide program logic

Reply:
Lets move reading from file and write to file to separate methods